packages/client/src/comp/states/Base.tsx

```diff
@@ -81,7 +86,8 @@ const ContainerlessBase = <Info extends StateInfo>({ enrichValues, children }: P
       otherCities,
       latLong,
       oid,
-      name: `${fields.firstName.value} ${fields.lastName.value}`,
+      name:
```

Comment:
This should be a special function.  You should consider the proper spacing if the middle name is empty or if the suffix is empty (i.e. no comma):

`(middleName ? ' ${middleName.trim()} ' : '')`